src/Processors/MergeSwagger.php

```diff
@@ -23,6 +23,7 @@ public function __invoke(Swagger $swagger)
                 $definitions = $annotation->definitions;
                 unset($annotation->paths);
                 unset($annotation->definitions);
+                $unmerged = array_merge($unmerged, $annotation->_unmerged);
```

Comment:
There was a problem here: If the `Swagger` annotation being merged also contained unmerged annotations, those annotations were lost. This is the quick single line fix.

The whole merging process seems quite brittle and may be going away anyway with the world view objects and what not, so I'm not sure any additional effort should be spent here. I can at least add a test for this though.